contracts/near/eth-client/src/tests.rs

```diff
@@ -461,3 +461,599 @@ fn predumped_block_can_be_added() {
fab6","0xd44cc3f552d0919fdc53f46b27199756","0x9d63eebec7612a639580daf176760942","0xc150a4428ac36990fe3e5ae2e980316e","0x229c8b77a95e3356af2917b4435eb627","0x04f9777e8983ea9b3692abe11b30eab0","0x4d04df6a53066fba9923e58ea8ec691a","0x8e3759bb5670e57fd0c69b76866f04a8","0x4a4eba100ca7f2ce9829bb4a7ecbdb7c","0x243b49f30dc3759c32a60d33e709c745","0xac10cbd730719f1478ad11c915c284dc","0x555b1247b929c14efbc92c3841ccc10c","0x158f1e4ac3fbda78f466298defde22b2","0x30a53f479a2ca1539ca2a5a71afae445"]},{"dag_nodes":["0x9b7ada0835277e4d4280b8e0a64e522800805adab57a4e84e76492cf47d0d00cd2d747d1693f12f56ad569925d18c944576cc20556d1f32bc54744f7d8e6a631","0x13aa9efd8d17de9a6bc0107256ec0fd63d7cf88d1d1d21cf4d1a2d2b219b4a784dd14b0a942752da63814829abb03651f47e9a336baf33de30dff9be2158f434"],"proof":["0xf37ac0387bf6755b22209c7cd9043791","0x319a55f08188a16a29f902c9d8d6c2dd","0xcd0ff55e3d6acffccfb94c115bbe7502","0x966c02d9a6ddb11764acf7204c91af36","0x2857f2468ef3df79993d1415374070e0","0xbe465eaac962b34083a451ed0654354d","0xe3ba312817b05ecd9db272eccd7a2d93","0x921dc284b275709e05ba2329c56d74dd","0x1a23bf2f3a3a804c3673f70817ce9011","0x7347a6e9a418c8b7985963443799086c","0xa2ac736ef41bfed61da26202fb37d8e2","0xe1fe15e3f4d42ef4e7820a331702eab4","0x242a52b95718b7a838b4705b8c40c1c4","0x96c9142b99fb62521d6d2dca20ca805b","0x97d64cd8afd60e205d3eeb2017fff49f","0x10a43c0161ee8f52ea711d623ba517b4","0xf8301fa0f368165cb6e81cab85c34373","0x6b351a26be2b3cc71f1ace8517d5d913","0xaf736f17d1f2390a1032455b17a3f9c1","0x6e84bab9db2db83dfeeab391dc776f97","0x510c76b59d871dfe77c9ddc364af4d50","0x685ecbd57edce902106c007682f5dce4","0x92470f1cc73f3c8c69875f0f4d3102df","0x4c9f091f9c5588d4b2dc2baf68287e07","0x147ee0ebf0aae1deebc019ab9b28a2d2","0x30a53f479a2ca1539ca2a5a71afae445"]},{"dag_nodes":["0x19246b61d66b5f8022c6faa294c30e8e6fc0bab83872571033b9061463a3f684deb16966089e80a19ad586f9cfae6d4f2d77bd74b980f53d5be718d7e753afba","0x6640cab3ca71b0247ec3906eb9ea574dfff53ac75a9821920956595c01ff018d9a706c2644561eb6f5a1145e6a95d25719da16efd0060b1722d1d2d5d1644a39"],"proof":["0xd0ec45344a7091b405f1d2c962a3f5f3","0x7601ef319f9054567f0e859629443a41","0x0ac36d658461a7cd07de8894aef3a76d","0xc434344d03c524e546fbc79e6ca2ff7e","0x18077ed4f9fc888dfcc2fca5d89cff2c","0x7ee6112533a50b3bee77e21f25f19a48","0x9453be8fabaa453f889380ef22551953","0x50a97983dabc2af2c97834307bc56928","0x7ffa4a53e14df7b7fb18adfe6e9f1481","0x5c9158107d37c248a221ea1eff8a1498","0x1c53d206c610aecc9be2b9f0bf885ef4","0x2899b5ed788d33e08b7d4ff4e25cfb3e","0xa3082f77cb0294af5f986a08e7788fa3","0x7af554433b5c569e313b6dac844ad88a","0x371079722c88b2b02518b2a3433aaa32","0xe49a9195c0dc650906536d868fd15ba9","0x69e6092f996258aaa973a13e6ca20af8","0x59d640825d9feb78e8f6286409ff4275","0xe4318aeb26c130ef204e892a35995dc5","0xdc6f2fe4cb45c7fae9f39f4cc4c2eece","0x29f2e5c1b9bb174b88b63b5d6b8d105b","0x914b93e809e85da6f6af72afe6233e9f","0xcfd16af046f96eb2269c21c16498a65e","0x10b71b7523c69d2b5e95bfb0d90a17ab","0x158f1e4ac3fbda78f466298defde22b2","0x30a53f479a2ca1539ca2a5a71afae445"]}]"#;
```

Comment:
This may use `json!` macro from `serde_json` crate.

Reply:
moved data to separate json file